### PR TITLE
CI/CD: change the time-consuming action to weekly and remove PPML-PySpark-K8s-Example-On-Gramine

### PR DESCRIPTION
**Pull request by ShanSimu**
## Description

change the time-consuming action to weekly.
remove PPML-PySpark-K8s-Example because it is contained in PPML-Spark-K8S-Example-On-Gramine.

### CODE REVIEW

File: .github/workflows/weekly_test.yml [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: create workflow badge
+      if: ${{ always() }}
+      uses: ./.github/actions/create-job-status-badge
+      with:
+        secret: ${{ secrets.GIST_SECRET }}
+        gist-id: ${{env.GIST_ID}}
+        file-name: nightly-test.json
```
Comment (liu-shaojun) [inline .github/workflows/weekly_test.yml:120]: the file-name should be weekly-test.json

File: .github/workflows/weekly_test.yml [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        - PPML-Occlum-k8s-ExampleTests
+        - PPML-Spark-K8S-Example-On-Gramine
+      tag:
+        description: 'docker image tag'
+        required: true
+        default: 'latest'
+        type: string
+
+env:
+  GIST_ID: bc8a699b455bced4a1aef138ad5df07e
```
Comment (liu-shaojun) [inline .github/workflows/weekly_test.yml:27]: please update the gist-id to 5c25555055d45e2de7849bca1618f53c, which is used for weekly-test workflow

**Review by liu-shaojun — APPROVED** [review]

**Review by Hanyu-Jin — APPROVED** [review]
LGTM

**Review by zzti-bsj — APPROVED** [review]
LGTM

**Pull request merged**